composer.json

```diff
@@ -30,6 +30,7 @@
         "cache/array-adapter": "^1.0",
         "cache/simple-cache-bridge": "^1.0",
         "cache/void-adapter": "^1.0",
+        "fakerphp/faker": "^1.23",
```

Comment:
Please revert. That should be part of the `composer.json` of your provider, not our main `composer.json`.

Reply:
@jbelien It's done!